src/main/java/io/getunleash/metric/ClientMetrics.java

```diff
@@ -1,5 +1,6 @@
 package io.getunleash.metric;
 
+import io.getunleash.engine.MetricsBucket;
```

Comment:
Probably not needed

Reply:
Its a bit sneaky but this was a drop in replacement for the existing metrics bucket. All I needed to do was include the new import and the code is unchanged